components/wb-fieldflow/_base.scss

```diff
@@ -5,6 +5,29 @@
  *
 */
 
+.wb-fieldflow {
+	&.gc-font-2019 {
+		&, & ~ .wb-fieldflow-form, & + div {
+			&, label, fieldset legend, input[type="submit"], select, .wb-fieldflow-label ~ p {
+				font-size: 20px;
+				line-height: 1.65em;
+			}
+
+			fieldset legend {
+				margin-bottom: 11.5px;
+			}
+
+			input[type="checkbox"], input[type="radio"] {
+				margin-top: 11px;
+			}
+
+			[type="submit"], select {
+				padding: .54em 1em;
+			}
+		}
+	}
+}
+
```

Comment:
Move this code in the GCWeb and here add a comment about it with a link to site/_defaults.scss